tsconifg.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "noImplicitAny": false,
+    "jsx": "react",
+    "baseUrl": "src",
```

Comment:
This repo doesnt have an src directory - instead it has utilities, client (and in PR server) as top level directories. Should we have a config file for client (ie browser) code, and one for server (Node) based code?